maplibre/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+//! # Maplibre-rs
+//!
+//! A multi-platform library for rendering vector tile maps with WebGPU.
```

Comment:
We need to extend this in the future. We maybe can put a description in the book and reference it from here via:

```
#![doc = include_str!("description.md")]
```

What do you think?

Reply:
Sounds good! Probably simpler than including files.